flow/cmd/handler.go

```diff
@@ -283,7 +283,7 @@ func (h *FlowRequestHandler) dropFlow(
 		}
 	}
 
-	dropFlowWorkflowID := fmt.Sprintf("%s-dropflow-%s", flowJobName, uuid.New())
+	dropFlowWorkflowID := flowJobName + "-peerflow"
 	workflowOptions := client.StartWorkflowOptions{
```

Comment:
setting conflict policy to reuse, & removing uuid, could work in #3549 but with this PR the stupid scenario where drop flow is created while cdc flow still running makes me hesitant to have it return a handle to a running cdc flow